test/langtools/tools/javac/doctree/ValueTest.java

```diff
@@ -102,6 +105,43 @@ int trailing_junk() { }
 ]
 */
 
+    /**
```

Comment:
Consider adding the following cases:
  * erroneous format
  * valid format and reference followed by the trailing "junk"

Reply:
done